ignite/cmd/bubblemodel/testnet_multi_node.go

```diff
@@ -0,0 +1,226 @@
+package cmdmodel
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"syscall"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+
+	"github.com/ignite/cli/v29/ignite/services/chain"
+)
+
+type NodeStatus int
+
+const (
+	Stopped NodeStatus = iota
+	Running
+)
+
+var _ tea.Model = MultiNode{}
+
+type MultiNode struct {
+	ctx  context.Context
```

Comment:
```suggestion
```

Is not used,  and the context should pass to the function, not be held by the struct